### DISCUSSION TITLE
How to resample wav files?

### DISCUSSION THREAD

**Discussion by joaquinjsb** [baresip/baresip#2868] [Q&A] [answered]
Hello,
I've been having issues with files with different rates and channels, how can I overcome this without having to manually change them via ffmpeg command line?

I got this error for example:

```audio: aufilter encode: Operation not supported [95]
audio: srate/ch of frame 22050/1 vs audio codec 8000/1. Use module auresamp!
resample: auresamp_setup error (Operation not supported [95])
audio: aufilter encode: Operation not supported [95]
audio: srate/ch of frame 22050/1 vs audio codec 8000/1. Use module auresamp!
```

but I enabled module resample, what else should I do?
thanks

**Comment by cspiel1** [comment, marked as answer]
See `re/rem/auresamp/resamp.c:244`:

```c
		if (irate % orate)
			return ENOTSUP;
```

Downsampling is only supported if output sample rate is a divisor of input sample rate.

The warning looks okay : "audio: aufilter encode: Operation not supported [95]"

**Reply by joaquinjsb** [reply]
thanks for letting me know this information
